bentoml/_internal/frameworks/tensorflow_v1.py

```diff
@@ -250,58 +162,64 @@ def load(
             " the functionalities of the given model, set `load_as_wrapper=False`"
             " will return a SavedModel object."
         )
-        assert all(
-            i is not None for i in [hub, resolve, native_module]
-        ), MissingDependencyException(
-            "`tensorflow_hub` is required to load a tfhub module."
-        )
+
+        if hub is None:
```

Comment:
do we need tf_hub here, since we are just loading the model from model store?

Reply:
we do, because if the model is imported from tfhub we need to know that